tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -12,20 +12,44 @@
 package alluxio.shell.command;
 
 import alluxio.AlluxioURI;
+import alluxio.Configuration;
+import alluxio.PropertyKey;
 import alluxio.client.WriteType;
 import alluxio.client.file.FileSystemTestUtils;
 import alluxio.exception.AlluxioException;
+import alluxio.security.GroupMappingServiceTestUtils;
+import alluxio.security.authentication.AuthType;
+import alluxio.security.group.provider.IdentityUserGroupsMapping;
 import alluxio.shell.AbstractAlluxioShellTest;
+import alluxio.util.CommonUtils;
 
 import org.junit.Assert;
 import org.junit.Test;
 
 import java.io.IOException;
+import java.util.List;
+import java.util.Random;
 
 /**
  * Tests for chown command.
  */
 public final class ChownCommandTest extends AbstractAlluxioShellTest {
+  private void checkFileOwnerAndGroup(String path, String expectedOwner, String expectedGroup)
```

Comment:
We usually follow one of two styles for private helper methods in tests

1. If the helper method is used by only one or two consecutive tests, we may put the helper method immediately after the tests that use it.
2. Otherwise, we put helper methods after all the tests.

We put private helper methods after tests because a person reading the file will want to see the tests first, not the private helper methods